### PR TITLE
Added a mention to the @Security annotation

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | 2.6+ |
| Fixed tickets | - |

### CODE REVIEW

File: cookbook/security/remember_me.rst [hunk 1]
```diff
@@ -175,21 +175,25 @@ In the following example, the action is only allowed if the user has the
...
     /**
-     * @Secure(roles="IS_AUTHENTICATED_FULLY")
+     * @Security("has_role('IS_AUTHENTICATED_FULLY')")
      */
     public function editAction($name)
     {
         // ...
     }
 
+.. versionadded:: 2.4
```
Comment (WouterJ) [inline cookbook/security/remember_me.rst]: as 2.4 was deprecated when 2.6 was released, we do not show any 2.4 versionadded in the 2.6 docs.

Comment (javiereguiluz) [inline cookbook/security/remember_me.rst]: Thanks for reporting this error. I've removed this `versionadded` directive.

**Comment by timglabisch** [comment]
:+1:

**Comment by stof** [comment]
should be applied to 2.5+ (the ``@Security`` annotation of FrameworkExtraBundle is not compatible with Symfony 2.3)

**Comment by xabbuh** [comment]
We don't merge into the `2.5` branch (it reach EOM at the end of January). So 2.6 is right.

**Comment by weaverryan** [comment]
Nice catch on this Javier - thanks!

**Pull request closed without merging**